custom/enikshay/integrations/ninetyninedots/repeater_generators.py

```diff
@@ -53,10 +74,11 @@ class PatientPayload(jsonobject.JsonObject):
     state_code = jsonobject.StringProperty(required=False)
     district_code = jsonobject.StringProperty(required=False)
     tu_code = jsonobject.StringProperty(required=False)
-    phi_code = jsonobject.StringProperty(required=False)
+    phi_code = jsonobject.StringProperty(required=False, exclude_if_none=True)
+    he_code = jsonobject.StringProperty(required=False, exclude_if_none=True)
```

Comment:
"health establishment". It is the private sector equivalent of a PHI (although is slightly different and lives in a different place in the location hierarchy)